### PR TITLE
Mymaps improved element naming

### LINKED ISSUE #921: Give the drawn features a relevant title
Currently, the drawn features are automatically given "element xx" as title.
It would be better in terms of usability to give them a more concrete name such as "Line 1" or "Point xx".

Also the counter should take the already drawn features in case of a loaded map. Currently, the counter starts from "0" even if features are already in the map. It may results in two features having the same title ( "element 1" for example).

### PR DESCRIPTION
**Pull request by petzlux**
fix #921

### CODE REVIEW

**Comment by petzlux** [comment]
Review please

File: geoportailv3/static/js/draw/drawdirective.js [hunk 1]
```diff
@@ -366,7 +373,33 @@ app.DrawController.prototype.onDrawEnd_ = function(event) {
...
+    case 'Polygon':
+      name = this.translate_.getString('Polygon');
+      break;
+  }
+  feature.set('name', name + ' ' + (++this.featureSeq_));
+  this.drawnFeatures_.features.forEach(function(drawnFeature) {
+    if (feature.get('name') === drawnFeature.get('name')) {
+      feature.set('name', name + ' ' + (++this.featureSeq_));
+    }
+  }, this);
```
Comment (pgiraud) [inline geoportailv3/static/js/draw/drawdirective.js]: What about initializing the sequence to the number of features + 1?
What I mean is that we could give the new feature a name with an index corresponding to the number of features already drawn.
```
feature.set('name', name + ' ' + this.drawnFeatures_.getCollection().getLength() + 1);
```
This is how google mymaps works.
Then, we can get rid of `featureSeq`.

File: geoportailv3/static/js/draw/drawdirective.js [hunk 2]
```diff
@@ -366,7 +373,33 @@ app.DrawController.prototype.onDrawEnd_ = function(event) {
...
+      break;
+    case 'LineString':
+      name = this.translate_.getString('LineString');
+      break;
+    case 'Polygon':
+      name = this.translate_.getString('Polygon');
+      break;
+  }
+  feature.set('name', name + ' ' + (++this.featureSeq_));
+  this.drawnFeatures_.features.forEach(function(drawnFeature) {
```
Comment (pgiraud) [inline geoportailv3/static/js/draw/drawdirective.js]: The `features` should be private in `drawnfeaturesservice` and we should use `getCollection` instead.

File: geoportailv3/static/js/draw/drawdirective.js [hunk 3]
```diff
@@ -366,7 +373,33 @@ app.DrawController.prototype.onDrawEnd_ = function(event) {
...
+        name = this.translate_.getString('Point');
+      }
+      break;
+    case 'LineString':
+      name = this.translate_.getString('LineString');
+      break;
+    case 'Polygon':
+      name = this.translate_.getString('Polygon');
+      break;
+  }
```
Comment (pgiraud) [inline geoportailv3/static/js/draw/drawdirective.js]: Couldn't this be simplified to the following?
```
var name;
var type = feature.getGeometry().getType();
if (type == 'Point' && this.drawLabel.getActive()) {
  name = this.translate_.getString('Label');
} else {
  name = this.translate_.getString(type);
}
```

Comment (petzlux) [inline geoportailv3/static/js/draw/drawdirective.js]: The explicit listing of the feature types has more to do with allowing transifex to collect the strings for the different geom types .
with `this.translate_.getString(type)` gettext wont be able to collect all the possible geom type strings.

Comment (pgiraud) [inline geoportailv3/static/js/draw/drawdirective.js]: Good point!
Thanks for the explanation.

**Comment by petzlux** [comment]
Okay addressed code review , will merge once Travis is happy

**Pull request merged**